subworkflows/local/deepvariant_caller.nf

```diff
@@ -5,20 +5,40 @@
 params.deepvariant_options = [:]
 
 include { DEEPVARIANT } from '../../modules/local/deepvariant/main'  addParams( options: params.deepvariant_options )
+include { GLNEXUS } from '../../modules/nf-core/modules/glnexus/main'  addParams( options: params.deepvariant_options )
 
 workflow DEEPVARIANT_CALLER {
     take:
         bam // channel: [ val(meta), path(bam), path(bai) ]
         fasta // path(fasta)
         fai // path(fai)
+        sample // channel: [ sample, sex, phenotype, paternal_id, maternal_id, case_id ]
 
     main:
         DEEPVARIANT ( bam, fasta, fai )
+        def file_list = DEEPVARIANT.out.gvcf.collect{it[1]}.toList()
```

Comment:
@ramprasadn `def` can be omitted here as this isn't needed when defining a new channel. You could use this syntax when declaring a function though :)